sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/compiler/QueryInterfaceGenerator.kt

```diff
@@ -32,15 +33,29 @@ class QueryInterfaceGenerator(val query: NamedQuery) {
         .addModifiers(DATA)
         .addSuperinterface(ClassName(query.select.sqFile().packageName, query.name.capitalize()))
 
+    var propertyPrints = listOf<String>()
     val constructor = FunSpec.constructorBuilder()
 
     query.resultColumns.forEach {
       typeSpec.addProperty(PropertySpec.builder(it.name, it.javaType, OVERRIDE)
           .initializer(it.name)
           .build())
       constructor.addParameter(it.name, it.javaType, OVERRIDE)
+
+      propertyPrints += "  ${it.name}: ${"$"}${it.name}"
```

Comment:
Should be pretty easy to add something to check if `it.javaType` is an `Array` or one of the 8 primitive arrays and change the logic to `${it.name.contentToString()}` but I don't know how to emit the import for the extension. Did we ever get member references in KotlinPoet?